tplus/client/base.py

```diff
@@ -65,7 +66,9 @@ async def _request(
     ) -> dict[str, Any]:
         relative_url = endpoint if endpoint.startswith("/") else f"/{endpoint}"
 
-        if not relative_url.startswith("/nonce") and not relative_url.startswith("/auth"):
+        if self.AUTH and (
```

Comment:
I want to make I am not breaking anything with this change-- I can't connect to the CE without disabling auth since those endpoints don't exist on the CE